### DISCUSSION TITLE
Browser zeigt seit neuester Version nichts mehr an am PC.

### DISCUSSION THREAD

**Discussion by DarnokHillstar** [beabel/radiowecker#39] [Fragen & Antworten / Q&A] [answered]
Hi beabel (kunigunde),

seit der neuesten Version funzt die Browseranzeige nicht mehr. Ich kann mir keinen Reim darauf machen.
- Chrome und Edge ausprobiert; - daran liegt es nicht.
- Beim Hochladen einer alten Version passiert mir das Gleiche jetzt. Warum auch immer.
- Beim der Eingabe der IP oder "radiowecker/" in der Eingabezeile des Browsers muss er offensichtlich was finden. Zeigt aber nur einen weißen Bildschirm an. Im Google DevTool ist es auch offensichtlich, dass nichts angezeigt wird:
hier den Code:- <html><head></head><body></body></html> -> somit kann er nichts anzeigen.

Ach ja:
Bei einem Test mit einem komplett neuen ESP32 ist über die radioconfig sehr wohl eine Browseranzeig erschienen. Somit konnte ich die Routerdaten eingeben. Danach aber wieder das gleiche Bild, wie oben beschrieben.

Was denke ich falsch?

Danke

Konrad

**Comment by Proxima-7** [comment, marked as answer]
Hallo nochmal an alle,
konnte das Problem der leeren Browser-Daten bei mir gerade lösen:
ich hatte in der Arduino IDE noch eine leicht abweichende Board-Url unter Preferences stehen: 
https://dl.espressif.com/dl/package_esp32_index.json
Diese funktioniert zwar auch, kann aber in den Tool-Settings nicht zwischen core 0 und core 1 trennen (möglicherweise laufen dann die Interrupts zusammen mit dem Arduino-Code auf nur einen Core ?).
Die korrekte Board-URL für den ESP32 Dev-Kit C V4 ist natürlich, wie im Projekt angegeben:
https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_index.json
Dort bitte unter Tools einstellen:
Events run on: core 0
Arduino runs on: core 1
Erase all flash before upload: enabled

PS: hoffe, es löst das Problem auch bei den anderen ..   ;-)

Gruß
Gerald

**Reply by TechZ-S** [reply]
Super! Genau das war's bei mir!
Vielen Dank!

**Reply by DarnokHillstar** [reply]
Hallo Gerald - Proxima-7

Du bist einfach nur SPITZE!
Das ist die Lösung des Problems! ALLES läuft!

Hinzufügen sollte man, - dass man nach der Eingabe der korrekten Board URL und ok drücken, die Arduino IDE nochmal verlassen muss und neu laden (hat mich nochmal 3 Hochladen gekostet, bis ich gemerkt habe, dass das ein Thema sein kann).

Übrigens habe ich als Board den DOIT ESP32 DEVKIT V1 angegeben (habe ich sonst immer so gemacht). Problemlos auch hier.

VIELEN DANK

Konrad
